recipes-kernel/linux/linux-imx-README

```diff
@@ -0,0 +1,6 @@
+The linux-imx kernel build uses a defconfig maintained in the layer. That
+defconfig is a copy of a defconfig maintained by NXP in the source.
+
+| defconfig in layer              | copied from NXP kernel source     |
+| linux-imx/imx-nxp-bsp/defconfig | arch/arm/configs/imx_v7_defconfig   |
+| linux-imx/mx8-nxp-bsp/defconfig | arch/arm64/configs/imx_v8_defconfig |
```

Comment:
I like the idea but it ought to be inside the `linux-imx_xx.bb` file and not a new one. It belongs to the recipe.